server/src/test/java/org/opensearch/extensions/ExtensionResponseTests.java

```diff
@@ -0,0 +1,51 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ */
+
+package org.opensearch.extensions;
+
+import org.opensearch.common.bytes.BytesReference;
+import org.opensearch.common.io.stream.BytesStreamInput;
+import org.opensearch.common.io.stream.BytesStreamOutput;
+import org.opensearch.test.OpenSearchTestCase;
+
+public class ExtensionResponseTests extends OpenSearchTestCase {
+
+    public void testExtensionBooleanResponse() throws Exception {
+        boolean response = true;
+        ExtensionBooleanResponse booleanRsponse = new ExtensionBooleanResponse(response);
```

Comment:
Small typo - booleanResponse

Reply:
Due to supply chain constraints, there's a vowel shortage.